includes/html/table/edit-ports.inc.php

```diff
@@ -62,6 +65,12 @@
         $checked = 'checked';
     }
 
+    $port_group_field = '<select class="input-sm" name="port_group_' . $port['port_id'] . '[]"  data-device_id="' . $port['device_id'] . '" multiple>';
```

Comment:
You set up a select2 backend controller, but don't use it here.  I suggest you use it :)
Only include the selected value here instead of all and then initialize the select2 widget on the frontend (enabling the tags option of select2)

Reply:
kinda confused :-)

Reply:
Search for `init_select2` javascript function

Reply:
$_port = Port::where('port_id', '=', $port_id);
$_port->groups()->sync(port_group_ids);

won't work

also with port_group_ids:
[
[port_group_id => 2],
[port_group_id => 1],
[port_group_id => 3],
]


what do i wrong :S

Reply:
i know what you mean, but how to use outside of blade framework with it's variables?

Reply:
init_select2()  is purely javascript.  You can use it anywhere.

Reply:
ok, tooked a look on it.
doing it with init_select2()  means a init_select2 per Port ... not sure if this is such good in handling

Reply:
@murrant  ?

Reply:
select2 loads the values from the backend so intializing 500 or more even should not be an issue.  The user can only click on one at a time.

Reply:
sure, but if i understand it right i have a select2 command for each port .

so code would look like:

```
...
select2( ...
select2( ...
select2( ...
select2( ...
...

```
not such fancy i think :D

Reply:
wouldn't it be in a foreach loop?